opensrp-chw/src/nacp/assets/json.form/hivst_mobilization_session.json

```diff
@@ -134,47 +134,47 @@
         "openmrs_entity": "concept",
         "openmrs_entity_id": "male_condoms_issued",
         "type": "edit_text",
-        "hint": "Male condoms issued",
+        "hint": "Idadi ya kondomu za kiume zilizotolewa",
         "v_numeric": {
           "value": "true",
-          "err": "Please enter a valid number"
+          "err": "Tafadhali ingiza numbari sahihi"
         },
         "v_min": {
           "value": "0",
-          "err": "The number must be equal or greater than 0"
+          "err": "Tarakimu ulioingiza inatakiwa iwe sawa au kubuwa kuliko 0"
         },
         "v_required": {
           "value": true,
-          "err": "Please enter the number"
+          "err": "Tafadhali ingiza nambar"
         }
       },
       {
         "key": "kit_used_in_demonstration",
         "openmrs_entity_parent": "",
         "openmrs_entity": "concept",
         "openmrs_entity_id": "kit_used_in_demonstration",
-        "label": "Was a kit used for demonstration in this session?",
+        "label": "Je kitepe kilitumika kwajili ya kufundishia",
         "text_color": "#000000",
         "type": "native_radio",
         "options": [
           {
             "key": "yes",
-            "text": "Yes",
+            "text": "Ndio",
             "openmrs_entity_parent": "",
             "openmrs_entity": "concept",
             "openmrs_entity_id": "yes"
           },
           {
             "key": "no",
-            "text": "No",
+            "text": "Hapana",
             "openmrs_entity_parent": "",
             "openmrs_entity": "concept",
             "openmrs_entity_id": "no"
           }
         ],
         "v_required": {
           "value": "true",
-          "err": "Please select an answer"
+          "err": "Tafadhali changuwa jibu "
```

Comment:
These need to be in opensrp-chw/src/nacp/assets/json.form-sw/hivst_mobilization_session.json